src/tracers.fs

```diff
@@ -0,0 +1,45 @@
+namespace Elmish.Tracers
```

Comment:
Let's get rid of `Tracers` namespace, as it's redundant given the module `Tracers` inside.